content/1_General/IO.mdx

```diff
@@ -547,9 +555,9 @@ You must rename the `.in` and `.out` files depending on the problem. For example
 
 <CPPSection>
 
-### Method 1 - [`freopen`](https://www.cplusplus.com/reference/cstdio/freopen/)
+### Method 1 - `freopen`
 
```

Comment:
no link to cplusplus.com? (it's fine either way; just wanted to make sure it was intentional)

same thing for <cstdio> below